### DISCUSSION TITLE
Variable not visible in output path

### DISCUSSION THREAD

**Discussion by jpgundrum** [rfennell/AzurePipelines#1459] [General]
Hello,

I am trying to use a tag variable to be apart of the output file path so the release note's name I generate is based on the tag version. However when I go to my posted artifact there is no value that gets placed at that location. The variable was set correctly, and I confirmed this by printing out the variable in a bash script. Does the field **outputfile** not take user set args correctly? Is there something wrong I am doing on my end? 

Thanks for the help, below are images illustrating my issues.

![image](https://user-images.githubusercontent.com/91096990/211663452-c4771430-70ff-4497-bbf7-3de19508addb.png)

![image](https://user-images.githubusercontent.com/91096990/211664834-e4b9aeb0-0db6-431d-8443-265f91dc0536.png)

**Comment by rfennell** [comment]
Run the pipeline with detailed logging (via the checkbox or set a variable `system.debug=true`) and check what path is being generated..

All I can assume is `$(tagvalue)` is being expanded by the pipeline process, tis should be done before any tasks are run
